packages/openapi-generator/src/model.ts

```diff
@@ -0,0 +1,146 @@
+import { kebabCase, last, pascalCase, unique } from '@sap-cloud-sdk/util';
```

Comment:
this is moere a `schema-util` instead of a model?

Reply:
yeah you're right. I tried to avoid "util" but it is what it is, why lie about it ;)